initfiles/componentfiles/configxml/roxie.xsd.in

```diff
@@ -77,6 +77,42 @@
                 </xs:appinfo>
               </xs:annotation>
             </xs:attribute>
+             <xs:attribute name="protocol" type="xpathType" use="optional" default="native">
```

Comment:
Also, are these XSD entries driving the Config mgr ui? if not, what's their purpose?